src/session.ts

```diff
@@ -464,12 +468,58 @@ async function setLocalHold(session: WebPhoneSession, flag: boolean): Promise<an
 }
 
 /*--------------------------------------------------------------------------------------------------------------------*/
+function getIncomingInfoContent(this: WebPhoneSession, request): any {
+    if (!request || !request.body) {
+        return {};
+    }
+    let ret = {};
+    try {
+        ret = JSON.parse(request.body);
+    } catch (e) {
+        return {}
+    }
+    return ret;
+}
+
+function sendMoveResponse(this: WebPhoneSession,
+                          reqId: number,
+                          code: number,
+                          description: string,
+                          options: any = {}) {
+    const extraHeaders = (options.extraHeaders || [])
+            .concat(this.ua.defaultHeaders)
+            .concat(['Content-Type: application/json;charset=utf-8']);
```

Comment:
Use `[...(options.extraHeader), ...this.ua.defaultHeaders, 'Content-Type: application/json;charset=utf-8']`